specs/grafana_event_handling_in_monitoring_integration.adoc

```diff
@@ -0,0 +1,313 @@
+:imagesdir: ./images
+
+= Grafana Event Handling In Monitoring Integration
+
+Alerting system is essential to spot the problem and notify it to the corresponding user.
+The threshholds for alerts will be configured in grafana and the alerts will be raised by grafana.
+These alerts are captured by monitoring-integration module and notified to respective user via the notifier.
```

Comment:
This section looks more like changes suggested. Rather it should be introduction to the issue and in short.